.circleci/config.yml

```diff
@@ -0,0 +1,71 @@
+version: 2
+dependencies:
+  cache_directories:
+    - ~/.composer/cache
+    - ~/build/vendor
+    - ~/build/node_modules
+jobs:
+  build:
+    docker:
+      - image: pantheonsystems/documentation:circle
+    working_directory: ~/build
+    steps:
+      - checkout
+      - run:
+          name: Generate build
+          command: |
+            npm install
+            ./node_modules/.bin/grunt
+            /documentation/bin/sculpin generate --env=prod --quiet
```

Comment:
@alexfornuto why does `sculpin generate` get run in multiple places? Here and `deploy-multidev`/`deploy-live`

Reply:
It needs to be generated in the multidev script with certain views changed for the multidev environment name. With that in mind, I suppose I'll keep generation there and in live, and test removing it here.